docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
```

Comment:
any particular reason for version 2 vs 3? not too sure if there is a big difference